### DISCUSSION TITLE
Welcome to Dreamer Discussions!

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/Dreamer#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FuryFire2004** [comment]
I just have too much time and may or may not actually just be a cat stepping on a keyboard and basically writing Hamilton in the process.
